src/components/Counter/Counter.js

```diff
@@ -0,0 +1,41 @@
+import { Button, TextField , Typography} from "@mui/material";
+import React, { useState } from "react";
+import "./counter.css";
+
+const Counter = () => {
+  const [count, setCount] = useState(0);
+
+  return (
+    <div className="root">
+      <Typography variant="h4"  >Counter App</ Typography>
+
+      <h2 className={count > 0 ? "positive" : "negative"}>{count}</h2>
+      <div className="firstApp">
+        <Button
+          variant="contained"
+          color="error"
+          onClick={() => setCount(count - 1)}
+        >
+          -
+        </Button>
+
```

Comment:
we also need a descriptive text field for the user to note what the counter is about